test/CMakeLists.txt

```diff
@@ -446,6 +446,20 @@ if (TARGET TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_eh_thread DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_global_control DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_task DEPENDENCIES TBB::tbb)
+
+    if (TBB_FUZZ_TESTING)
+        if (NOT CMAKE_CXX_COMPILER_ID STREQUAL IntelLLVM)
+            message(FATAL_ERROR "Fuzzing requires IntelLLVM compiler.")
+        endif()
+        if (MSVC AND NOT (CMAKE_MSVC_RUNTIME_LIBRARY STREQUAL MultiThreaded))
+            message(FATAL_ERROR "Fuzzing on Windows requires CMAKE_MSVC_RUNTIME_LIBRARY to be set to MultiThreaded.")
+        endif()
+        tbb_add_test(SUBDIR tbb NAME test_fuzzing)
+        add_dependencies(test_fuzzing test_task)
+        target_compile_definitions(test_fuzzing PRIVATE CMD="$<TARGET_FILE:test_task>")
```

Comment:
Maybe change `CMD` to `TEST_EXECUTABLE`?

Reply:
I will add output redirects to the command, so `CMD` might be a better name.

Reply:
Done.